apollo-router/src/plugins/traffic_shaping/mod.rs

```diff
@@ -528,17 +604,25 @@ mod test {
         mock_service.expect_clone().returning(|| {
             let mut mock_service = MockSupergraphService::new();
 
-            mock_service.expect_call().times(0..2).returning(move |_| {
-                Ok(SupergraphResponse::fake_builder()
-                    .data(json!({ "test": 1234_u32 }))
-                    .build()
-                    .unwrap())
+            println!("cloning the supergraph");
```

Comment:
Should we keep it ?

Reply:
removed in https://github.com/apollographql/router/pull/2034/commits/28334c2ac432856dd039996d0926eefef6bc8f54